testapp/views.py

```diff
@@ -72,28 +72,8 @@ def info(request):
   data = [city, term]
   #yelp_call() returns dictionary of restaurant and its information
   restaurant_dic = yelp_call(term, city)
-
-  restaurant_info = ''
-  restaurant_name = restaurant_dic.get('name')
-  location_info = " "
-  location = restaurant_dic.get('location')
-  for info in location:
-  	if location[info] != '':
-	  	if info == 'display_address':
-	  		location_info += "Address: " + location[info][0] + "<br/>"
-	  		continue
-	  	location_info += str(info) + ": " + str(location[info]) + "<br/>"
-  phone = restaurant_dic.get('phone')
-  price = restaurant_dic.get('price')
-  rating = str(restaurant_dic.get('rating'))
-
-  restaurant_info = ("Restaurant name: " + restaurant_name + "<br/>"
-  					+ "Phone number: " + phone + "<br/>"
-  					+ "Rating: " + rating + "<br/>"
-  					+ location_info)
-
   save_user_request(data)
-  return HttpResponse(restaurant_info)
+  return render(request, 'basic.html', {'restaurant_dic': restaurant_dic})
```

Comment:
As a style thing, we can see we're making this a dictionary so no need to append `_dic` at the end.